GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
 # OpenRefine Governance Model
 
 ## Summary / Overview
-OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a plugin architecture and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.
+OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a [plugin architecture](https://openrefine.org/docs/technical-reference/writing-extensions) and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.
 
 ## Roles and responsibilities
-OpenRefine development is based on user consensus and open discussion between users. Decision making must be done in a transparent, open fashion (i.e. using OpenRefine's discussion list and issue list). No decisions about the project’s direction, bug fixes or features may be done in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
+OpenRefine is a meritocratic, consensus-based community project based on the [Apache Decision Making process](https://community.apache.org/committers/decisionMaking.html).
 
-All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
+Decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion list](https://forum.openrefine.org/c/dev/8) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No decisions about the project’s direction, bug fixes, or features may be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
 
-Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process. This document describes how that participation takes place.
+All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
 
 ### Users
```

Comment:
These sections create expectations around people and try to sort them into buckets of with little to no relevance to the governance. I would suggest limiting these sections to match those of GitHub contributors, it would simplify this a lot while staying true to the actual state of order.

Reply:
Your comment implies that this governance file should be limited in scope to just the GitHub organization. That may be one way to do it, but then I think we need another governance file covering the entire project/community and not just the parts happening on GitHub. While each of the files may be simplified, it will be harder to get an overview both of the actual state and the intended state.

Reply:
@Ainali you don't need to sort people and their work into buckets to have a governance document, removing these sections wouldn't change the policy at all.

I do argue for the governance document to be split as the current approach makes things unclear and the later sections contradicts the former ones.